Sources/Contour/Transport/RabbitMQ/Internal/Listener.cs

```diff
@@ -103,17 +105,17 @@ internal class Listener : IDisposable
         /// <param name="validatorRegistry">
         /// Реестр механизмов проверки сообщений.
         /// </param>
-        public Listener(IChannelProvider channelProvider, ISubscriptionEndpoint endpoint, RabbitReceiverOptions receiverOptions, MessageValidatorRegistry validatorRegistry)
+        public Listener(IBusContext busContext, IRabbitConnection connection, ISubscriptionEndpoint endpoint, RabbitReceiverOptions receiverOptions, MessageValidatorRegistry validatorRegistry)
         {
+            this.busContext = busContext;
+            this.channel = connection.OpenChannel();
```

Comment:
This change is going to be rolledback because it breaks the listener scaling mechanism.